### PR TITLE
Add new module: <InfoHeader/>

### PR DESCRIPTION
**Pull request by drewvolz**
Usage:
```js
<InfoHeader
    message="this is your message"
    title="this is your title"
/>
```

This generates the top message header that is used in the "report" views for Building Hours and Dictionary feedback reporting.

### CODE REVIEW

File: source/views/building-hours/report/overview.js [hunk 1]
```diff
@@ -203,13 +181,11 @@ export class BuildingHoursProblemReportView extends React.PureComponent<
...
-				<View style={styles.helpWrapper}>
-					<Text style={styles.helpTitle}>Thanks for spotting a problem!</Text>
-					<Text style={styles.helpDescription}>
-						If you could tell us what the new times are, we&rsquo;d greatly
-						appreciate it.
-					</Text>
-				</View>
+				<InfoHeader
+					message="If you could tell us what the new times are, we&rsquo;d greatly
+                        appreciate it."
```
Comment (hawkrives) [inline source/views/building-hours/report/overview.js]: Why is this on a newline?

File: source/views/dictionary/report/editor.js [hunk 2]
```diff
@@ -65,13 +65,11 @@ export class DictionaryEditorView extends React.PureComponent<Props, State> {
...
-				<View style={styles.helpWrapper}>
-					<Text style={styles.helpTitle}>Thanks for spotting a problem!</Text>
-					<Text style={styles.helpDescription}>
-						If you could tell us what the word and definition should be,
-						we&rsquo;d greatly appreciate it.
-					</Text>
-				</View>
+				<InfoHeader
+					message="If you could tell us what the word and definition should be,
+                        we&rsquo;d greatly appreciate it."
```
Comment (hawkrives) [inline source/views/dictionary/report/editor.js]: Same question about newlines

**Review by hawkrives — COMMENTED** [review]
?

File: source/views/building-hours/report/overview.js (hunk 1 above)
Comment (rye) [reply source/views/building-hours/report/overview.js]: I have changed this under the assumption that this was not intentional.

File: source/views/dictionary/report/editor.js (hunk 2 above)
Comment (rye) [reply source/views/dictionary/report/editor.js]: I have changed this under the assumption that this was not intentional.

**Review by rye — APPROVED** [review]
RSLGTM.

**Review by hawkrives — APPROVED** [review]

**Comment by drewvolz** [comment]
@rye thanks for cleaning up my unintentional newlines.

**Comment by drewvolz** [comment]
@hawkrives is there anything that I should look into as far as making this component adhere to app theming? Currently I've set a background color that doesn't come from the theming module.

**Comment by rye** [comment]
@drewvolz do you think that can get put in a separate PR? I'm feeling an itch to merge this and beta.

**Comment by drewvolz** [comment]
That can be separate. Feel free to merge.

**Comment by rye** [comment]
Thanks, merging.

**Pull request merged**